backend/hitas/services/validation.py

```diff
@@ -0,0 +1,17 @@
+import re
+
+from rest_framework.exceptions import ValidationError
+
+
+def validate_postal_code(postal_code: str) -> str:
+    match = re.match(r"^\d{5}$", postal_code)
+    if match is None:
+        raise ValidationError(f"{postal_code!r} is not a valid postal code.")
+    return postal_code
+
+
+def validate_quarter(value: str) -> str:
+    match = re.match(r"^\d{4}Q[1234]$", value)
```

Comment:
These are pretty simple regexes, but in general adding an example value still helps 😉

Reply:
Added Docstrings with example values.